templates/override-service.systemd.conf.j2

```diff
@@ -1,6 +1,4 @@
 [Service]
 {% for k, v in dnsdist_service_overrides.items() %}
-{% if k == 'ExecStart' %}ExecStart=
```

Comment:
afaik, systemd requires and empty `ExecStart` to clear the full value. Otherwise it appends. See `man systemd.unit`

> Note that for drop-in files, if one wants to remove entries from a setting that is parsed as a list (and is not a dependency), such as AssertPathExists= (or e.g. ExecStart= in service units), one needs to first clear the list before re-adding all entries except the one that is to be removed. Dependencies (After=, etc.) cannot be reset to an empty list, so dependencies can only be added in drop-ins. If you want to remove dependencies, you have to override the entire unit.

Reply:
Yep, that's correct and needs to be added back.

It also seems like we need the same thing to be done for the ExecStartPre to avoid that from being triggered multiple times with potentially incorrect parameters.